### PR TITLE
User can view list of flash cards

### PR DESCRIPTION
**Pull request by StevenNagata**
react-flashcards-issue3

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,25 +1,53 @@
...
+import Navbar from './nav-bar'
+import hash from './hash'
+import MyFlashcards from './my-flashcards'
 import FlashcardForm from './flashcard-form'
 
 export default class App extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
+      view: { path: 'view' },
```
Comment (thebearingedge) [inline src/app.js]: You should initialize `state.view` using the `location.hash` so refreshing the page still keeps you on the same screen.

File: src/my-flashcards.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+export default class MyFlashcards extends React.Component {
+  render() {
+    if (this.props.flashcards.length === 0) {
+      return (
+        <NoFlashcards takeToForm={this.props.takeToForm} />
+      )
+    }
+    else {
+      return (
+        <div className="container p-2">
```
Comment (thebearingedge) [inline src/my-flashcards.js:13]: This is a little weird having two containers nested like this. Any particular reason?

File: src/no-flashcards.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function NoFlashcard(props) {
+  return (
+    <div className="container-fluid d-flex justify-content-center">
+      <div>
+        <h3>You have no flash cards</h3>
+        <div className="container-fluid d-flex justify-content-center">
+          <a href="#create"><button onClick={props.takeToForm} type="button" className="btn btn-dark">Create flash cards</button></a>
```
Comment (thebearingedge) [inline src/no-flashcards.js]: If your `href` is set to the correct hash, the `window` will dispatch a `'hashchange'` event for you, so you don't need a click listener.

You can style `<a>` elements to look like buttons with Bootstrap (`className="btn btn-dark"`).

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by StevenNagata** [comment]
react-flashcards-issue3

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**